hubspot/serializers.py

```diff
@@ -146,8 +160,15 @@ def get_price(self, instance):
         """Return the latest product version price"""
         product_version = instance.latest_version
         if product_version:
-            return product_version.price.to_eng_string()
-        return "0.00"
+            return product_version.price
+        return 0
+
+    def get_description(self, instance):
+        """Return the latest product version price"""
+        product_version = instance.latest_version
+        if product_version:
+            return product_version.price
```

Comment:
return product_version.description

Reply:
this still needs to be changed to `return product_version.description`, as is the products will not sync due to a serializer error